requirements/base.in

```diff
@@ -1,9 +1,10 @@
 # Core requirements for using this application
+-c constraints.txt
 
 # Django/Framework Packages
 django<2,>=1.11
 django-model-utils>=2.3.1
-djangorestframework>=3.1,<3.7
+djangorestframework
```

Comment:
Should this be pinned by a major version number?

Reply:
The tests currently test all current versions and will automatically test the latest version so I feel pretty comfortable not to pin it here.  If the next major version breaks this app, the tests will pick it up on the automated `make upgrade` PR once the new version is available.  Pinning to the major version here would mean that we would need to come back and unpin here before we could upgrade edx-platform, and we would need to do the same in all apps that depend on DRF.  This felt too cumbersome to me.  Let me know if you disagree and we should change this strategy.